### DISCUSSION TITLE
Spectral library using Uniprot FASTA file

### DISCUSSION THREAD

**Discussion by pejota66** [vdemichev/DiaNN#712] [General]
Hello,

For the Creation of the spectral Library using Fasta digest, you advise just running DIA-NN adding the Fasta file without specifying any raw files and enabling the Deep learning-based spectra, RTs and IMs prediction option in the Precursor ion generation pane. 
Regarding the peptide length, precursor charge, precursor m/z and the settings on the right panel, should we also keep them as suggested? 

Many thanks.

**Comment by vdemichev** [comment]
Yes, exactly. Default values are best unless recommended otherwise in DIA-NN docs.
